src/main/resources/js/condec.knowledge.page.js

```diff
@@ -114,6 +126,11 @@
 			var node = tree.node.data;
 			treant.buildTreant(node.key, true, "");
 		});
+		
+		var selectedGroup = document.getElementById("select-decision-group-input").value;
+		if(!selectedGroup===undefined || selectedGroup != ""){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)